packages/block-template/src/block-protocol-types.ts

```diff
@@ -0,0 +1,258 @@
+/*
+ * This file will be replaced with a package import.
+ * Watch https://github.com/blockprotocol/blockprotocol for updates.
+ */
+
+import type { VoidFunctionComponent } from "react";
+
+export type BlockVariant = {
+  description?: string;
+  displayName?: string;
+  icon?: string;
+  properties?: JSONObject;
+};
+
+/**
+ * @todo type all as unknown and check properly
+ * we can't rely on people defining the JSON correctly
+ */
+export type BlockMetadata = {
+  author?: string;
+  description?: string;
+  displayName?: string;
+  externals?: Record<string, string>;
+  license?: string;
+  icon?: string;
+  name?: string;
+  schema?: string;
+  source?: string;
+  variants?: BlockVariant[];
+  version?: string;
```

Comment:
This should also include image `image?: string`

Reply:
Thanks! I’ll add `image?` before merging. I’ll add image field here and upstream before merging.